README.md

```diff
@@ -200,14 +200,14 @@ async fn run() -> Result<(), Error> {
         .allow_no_indices(true)
         .send()
         .await?;
-    
+
     // get the HTTP response status code
     let status_code = search_response.status_code();
-    
+
     // read the response body. Consumes search_response
-    let response_body = search_response.json::<Value>().await?; 
-    
-    // read fields from the response body         
+    let response_body = search_response.read_body::<Value>().await?;
```

Comment:
This should be `json` - see #84

Reply:
Hi @russcam , I found the issue. I was using the crates.io package:

`elasticsearch = "7.6.1-alpha.1"`
This did not have the new changes you had pushed and resulted in the following error which I thought was the bug:
`  --> src/main.rs:24:41
   |
24 |     let response_body = search_response.json::<Value>().await?;
   |                                         ^^^^ method not found in elasticsearch::http::response::Response`

When I chose the Github path instead:
`elasticsearch = {git = "https://github.com/elastic/elasticsearch-rs"}`
I was not receiving the error. 

So the crate can be updated to fix it.

Reply:
There are changes in master that are not yet published to crates.io, as there will be from time to time. Since the PR is against master, the PR should be updated in line with the changes against master.